app/uk/gov/hmrc/helptosave/controllers/StrideController.scala

```diff
@@ -24,16 +24,16 @@ import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent}
 import uk.gov.hmrc.auth.core.AuthConnector
 import uk.gov.hmrc.helptosave.config.AppConfig
-import uk.gov.hmrc.helptosave.connectors.PayePersonalDetailsConnector
+import uk.gov.hmrc.helptosave.connectors.DESConnector
 import uk.gov.hmrc.helptosave.repo.EnrolmentStore
-import uk.gov.hmrc.helptosave.services.EligibilityCheckService
+import uk.gov.hmrc.helptosave.services.HelpToSaveService
 import uk.gov.hmrc.helptosave.util.{LogMessageTransformer, WithMdcExecutionContext}
 import uk.gov.hmrc.helptosave.util.Logging._
 
-class StrideController @Inject() (val eligibilityCheckService:  EligibilityCheckService,
-                                  payePersonalDetailsConnector: PayePersonalDetailsConnector,
-                                  authConnector:                AuthConnector,
-                                  enrolmentStore:               EnrolmentStore)(implicit transformer: LogMessageTransformer, override val appConfig: AppConfig)
+class StrideController @Inject() (val helpToSaveService: HelpToSaveService,
+                                  dESConnector:          DESConnector,
```

Comment:
same as above

Reply:
I have removed the desConnector here as its not used, thanks.